lib/sequenceserver/routes.rb

```diff
@@ -65,8 +65,10 @@ class Routes < Sinatra::Base
         key: 'rack.session.sequenceserver',
         secret: ENV.fetch('SESSION_SECRET') { SecureRandom.alphanumeric(64) }
       )
-
-      use Rack::Csrf, raise: true, skip: ['POST:/cloud_share']
+      csrf_protected = ENV.fetch('CSRF_PROTECTED', 'true')
+      if csrf_protected == 'true'
```

Comment:
Ok if config is not loaded at this point let's just invert the logic in your originally proposed change so that CSRF protection is on by default:

```suggestion
      unless ENV['SKIP_CSRF_PROTECtION'] == 'true'
```